app/views/notifications/_comment.html.erb

```diff
@@ -38,7 +36,7 @@
         </a></b>
       <%= render "notifications/shared/comment_box", json_data: json_data, notification: notification, context: "moderation" %>
       <br>
-      <div class="footnote">All negative reactions are 100% private. Thank you for being a trusted <%= ApplicationConfig["COMMUNITY_NAME"] %> member.</div>
+      <div class="footnote">All negative reactions are 100% private.</div>
```

Comment:
I made some unrelated copy adjustments along with my PR while I was in notifications land.